api/hooks.go

```diff
@@ -140,10 +141,24 @@ func closeBody(rsp *http.Response) {
 	}
 }
 
-func triggerHook(event HookEvent, user *models.User, instanceID, jwtSecret string, hconfig *conf.WebhookConfig) error {
-	if hconfig.URL == "" {
+func triggerHook(event HookEvent, user *models.User, instanceID string, config *conf.Configuration) error {
+	if config.Webhook.URL == "" {
 		return nil
 	}
+	hookURL, err := url.Parse(config.Webhook.URL)
+	if err != nil {
+		return err
```

Comment:
Might be worth using an `errors.Wrapf` so we can distinguish the two URL parses apart